docs/src/modules/getting-started/pages/quick-deploy.adoc

```diff
@@ -17,6 +17,8 @@ akka auth login
 
 . Build a container image of your service:
 +
+NOTE: The `mvn install` command of the Maven Docker plugin connects to the locally running Docker daemon using the `DOCKER_HOST` environment value. Depending on your Docker tooling you might need to set this accordingly, you may use `docker context ls` to identify the correct value.
```

Comment:
FWIW, I do not have `DOCKER_HOST` in my env, but the command still works 🤷

Reply:
Apparently the Docker context gives the default.